examples/example-vite-grpc-playground/src/components/schema-explorer.tsx

```
@@ -0,0 +1,79 @@
+                        key={tableName}
+                        className="border border-zinc-800 rounded-lg"
+                    >
+                        <div
+                            className="flex items-center gap-2 p-2 cursor-pointer hover:bg-zinc-800"
+                            onClick={() =>
+                                setExpandedTables((prev) => ({
+                                    ...prev,
+                                    [tableName]: !prev[tableName],
+                                }))
+                            }
+                        >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)